### ISSUE TITLE
Weird graphical glitching after a few mins of playing makes game unplayable

### ISSUE THREAD

**Issue by Leew85** [OpenRCT2/OpenRCT2#19169]
### Operating System

Windows 10 

### OpenRCT2 build

OpenRCT2, v0.4.3 (285e0fc on master) provided by GitHub

### Base game

RollerCoaster Tycoon 2

### Area(s) with this issue?

This bug is a graphical glitch or error

### Describe the issue

After a few mins of playing online, the graphics go all glitchy and the game becomes unplayable.  This occurs on two different servers I have played (Mozers Casual I and Mozers FFA).
![rct_glitch1](https://user-images.githubusercontent.com/122828771/212757579-18f904bc-968b-4c66-ae85-ba96e913e74e.png)
![rct_glitch2](https://user-images.githubusercontent.com/122828771/212757582-c0d1e640-367d-4197-9e93-268fa45c3adb.png)

### Steps to reproduce

Open OpenRCT2
Join Mozers Casual I server
Scroll around for 5 mins admiring the park (no need to build anything or do anything else)
At a random point the graphics will glitch out like the 1st screenshot and the game will be unplayable.

**Comment by Gymnasiast** [comment]
We will need to know your graphics settings, could you paste the contents of `config.ini`?

**Comment by Leew85** [comment]
As requested.
Also worth noting that this only started happening after upgrading to version 4.3, I had never had this problem before that.

[general]
always_show_gridlines = true
autosave = 1
autosave_amount = 10
confirmation_prompt = false
currency_format = GBP
custom_currency_rate = 10
custom_currency_affix = SUFFIX
custom_currency_symbol = "Ctm"
edge_scrolling = true
edge_scrolling_speed = 12
fullscreen_mode = 0
fullscreen_height = 1080
fullscreen_width = 1920
rct1_path = "E:\\games\\Steam\\SteamApps\\common\\RollerCoaster Tycoon Deluxe"
game_path = "E:\\games\\Steam\\SteamApps\\common\\Rollercoaster Tycoon 2"
landscape_smoothing = true
language = en-GB
measurement_format = METRIC
play_intro = false
save_plugin_data = true
debugging_tools = true
show_height_as_units = false
temperature_format = CELSIUS
window_height = 904
window_snap_proximity = 5
window_width = 1665
default_display = 0
drawing_engine = OPENGL
uncap_fps = false
use_vsync = true
date_format = DD/MM/YY
auto_staff = true
handymen_mow_default = false
default_inspection_interval = 2
last_run_version = "0.4.3"
invert_viewport_drag = false
load_save_sort = DATE_DESCENDING
minimize_fullscreen_focus_loss = true
disable_screensaver = true
day_night_cycle = false
enable_light_fx = false
enable_light_fx_for_vehicles = false
upper_case_banners = false
disable_lightning_effect = false
steam_overlay_pause = true
window_scale = 1.000000
show_fps = true
multi_threading = false
trap_cursor = false
auto_open_shops = true
scenario_select_mode = 1
scenario_unlocking_enabled = false
scenario_hide_mega_park = true
last_game_directory = "E:\\lee\\Documents\\OpenRCT2\\save"
last_landscape_directory = ""
last_scenario_directory = "E:\\lee\\Documents\\OpenRCT2\\scenario\\"
last_track_directory = "E:\\lee\\Documents\\OpenRCT2\\track"
use_native_browse_dialog = false
window_limit = 64
zoom_to_cursor = true
render_weather_effects = true
render_weather_gloom = true
show_guest_purchases = false
show_real_names_of_guests = true
allow_early_completion = true
asset_pack_order = ""
enabled_asset_packs = ""
virtual_floor_style = GLASSY
transparent_screenshot = true
transparent_water = false
invisible_rides = false
invisible_vehicles = false
invisible_trees = false
invisible_scenery = false
invisible_paths = false
invisible_supports = false
last_version_check_time = 1673897842

[interface]
toolbar_show_finances = true
toolbar_show_research = true
toolbar_show_cheats = true
toolbar_show_news = true
toolbar_show_mute = true
toolbar_show_chat = true
toolbar_show_zoom = true
console_small_font = false
current_theme = "*RCT2"
current_title_sequence = "*OPENRCT2"
random_title_sequence = false
object_selection_filter_flags = 16383
scenarioselect_last_tab = 0
list_ride_vehicles_separately = false

[sound]
audio_device = ""
master_sound = true
master_volume = 62
title_music = 1
sound = true
sound_volume = 100
ride_music = true
ride_music_volume = 100
audio_focus = true

[network]
player_name = "VeloLee"
default_port = 11753
listen_address = ""
default_password = ""
stay_connected = true
advertise = true
advertise_address = ""
maxplayers = 16
server_name = "Server"
server_description = ""
server_greeting = ""
master_server_url = ""
provider_name = ""
provider_email = ""
provider_website = ""
known_keys_only = false
log_chat = false
log_server_actions = false
pause_server_if_no_clients = false
desync_debugging = false

[notifications]
park_award = true
park_marketing_campaign_finished = true
park_warnings = true
park_rating_warnings = true
ride_broken_down = true
ride_crashed = true
ride_casualties = true
ride_warnings = true
ride_researched = true
ride_stalled_vehicles = true
guest_warnings = true
guest_left_park = true
guest_queuing_for_ride = true
guest_on_ride = true
guest_left_ride = true
guest_bought_item = true
guest_used_facility = true
guest_died = true

[font]
file_name = ""
font_name = ""
x_offset = 0
y_offset = 1
size_tiny = 1
size_small = 0
size_medium = 0
size_big = 0
height_tiny = 0
height_small = 0
height_medium = 0
height_big = 0
enable_hinting = true
hinting_threshold = 0

[plugin]
enable_hot_reloading = false
allowed_hosts = ""

**Comment by Gymnasiast** [comment]
Does the problem occur when you change the rendering engine to `Software (hardware display)`?

**Comment by Leew85** [comment]
That seems to have sorted it, thanks 👍🏻

**Comment by Gymnasiast** [comment]
There has been one small OpenGL change between releases v0.4.2 and v0.4.3, which might be the culprit. I’m going to prepare a build for you that has this change reverted to see if that fixes the OpenGL problem. But we’re currently working on a big refactor, so it might take a little bit.
